app/labor/rate_limit_checker.rb

```diff
@@ -17,7 +18,10 @@ def limit_by_situation(situation)
              else
                false
              end
-    ping_admins if result == true
+    if result
```

Comment:
I suggest adding a test for the changed behavior here.

Reply:
added more spec